README.md

```diff
@@ -9,6 +9,10 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
 [![DOI](https://zenodo.org/badge/635090135.svg)](https://zenodo.org/doi/10.5281/zenodo.10034636)
 
+<p align="center">
+  <img width="300px" alt="logo" src="https://private-user-images.githubusercontent.com/51029046/483073703-d2bccfbd-19c9-4448-a5d9-6ac927c32cc5.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTYzNzMwNjcsIm5iZiI6MTc1NjM3Mjc2NywicGF0aCI6Ii81MTAyOTA0Ni80ODMwNzM3MDMtZDJiY2NmYmQtMTljOS00NDQ4LWE1ZDktNmFjOTI3YzMyY2M1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODI4VDA5MTkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTExMDQ2Y2UwMjhlOWRlY2Y2N2FlMzI1YTFjMmM2ZWMyMTRhNmE1NTc4NmY1ZmM0YjliZjE0NDVjMWI5MzkzMzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.hKVrNmVAv9DLOHVymMhjT5UUKnyGQzLiYZFyVlUedwQ">
```

Comment:
Is there any way to get a shorter URL?

Reply:
I don't know

Reply:
I'll see if I can find out how we do that in Trixi.jl.